packages/core/src/StateNode.ts

```diff
@@ -102,6 +103,8 @@ import {
 } from './stateUtils';
 import { createInvocableActor } from './Actor';
 import { toInvokeDefinition } from './invokeUtils';
+import { TypegenDisabled } from './typegenTypes';
+import { ServiceMap } from '.';
```

Comment:
Can this be imported from types? (nit)

Reply:
vscode autoimports are annoying at times :s i agree - we shouldn't ever import from `'.'`. It tends to introduce cycles in the dependency graph (and while they work, it's best to avoid them).